packages/components/src/unit-control/types.ts

```diff
@@ -28,7 +26,7 @@ export type WPUnitControlUnit = {
 	/**
 	 * Default value for the unit, used when switching units.
```

Comment:
This confused me for a bit, 👇  might've been easier to understand at first glance:

```suggestion
	 * Default value (quantity) for the unit, used when switching units.
```

Reply:
Makes sense, the use of the word "value" in this component is tricky and confusing